test/linalg/svd.jl

```diff
@@ -4,6 +4,31 @@ using Base.Test
 
 using Base.LinAlg: BlasComplex, BlasFloat, BlasReal, QRPivoted
 
+@testset "Simple svdvals / svdfact tests" begin
+    ≈(x,y) = isapprox(x,y,rtol=1e-15)
```

Comment:
Something like that might be neat. In the meantime though, I recommend
```julia
⩸(x, y) = isapprox(x, y, rtol=1e-15)
@test x ⩸ y
```
(that's `\equivDD`, use whatever you like... other than `≈` 😉)

Could also do
```julia
function with_rtol(f::Function, tol::Float64)
    ≈(x, y) = isapprox(x, y, rtol=tol)
    f()
end

@testset "..." with_rtol(1e-15) do
    @test x ≈ y
end
```